modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -32,7 +33,139 @@ With those steps completed, stop your webserver.
 sudo service apache2 stop
 ----
 
-[[backup-your-existing-installation]]
+== Backup Your Existing Installation
+
+First, xref:maintenance/backup.adoc[backup] the following items:
+
+.  The ownCloud server data directory
+.  The complete ownCloud directory
+.  The ownCloud server database
+
+You can find detailed steps to do backups here: xref:maintenance/backup.adoc[].
+
+[source,console]
+----
+
+# This example assumes Ubuntu Linux and MariaDB
+mv /var/www/owncloud /var/www/owncloud-old-version-number
+mysqldump -u<username> -p<password> <databasename> > <ownCloud-Version-Dump.sql>
+----
+
+== Review Third-Party Apps
+
+# Enable maintenance mode using the occ command.
+sudo -u www-data php occ maintenance:mode --on
+----
+
+In clustered environment please check that all nodes are in maintenance mode.
+== Stop the Webserver
+
+
+Review any installed third-party apps for compatibility with the new ownCloud release.
+Ensure that they are all disabled before beginning the upgrade.
+Third party apps are all apps that are not distributed by https://marketplace.owncloud.com/publishers/owncloud
+
+Then, disable the apps, via the command-line.
+ 
+[source,console]
+----
+# This command lists all apps by <app-id> and app version
+sudo -u www-data php occ app:list
+
+# This command disables the app with the given <app-id>
```

Comment:
`#` is a first level header = title header. Is this intended?

Reply:
'#' within a console block it is a comment